Northstar.CustomServers/mod/scripts/vscripts/mp/_score.nut

```diff
@@ -108,7 +108,7 @@ void function ScoreEvent_PlayerKilled( entity victim, entity attacker, var damag
 	if ( !attacker.IsPlayer() )
 		return
 
-
+	attacker.p.numberOfDeathsSinceLastKill = 0 // since they got a kill, remove the comeback trigger
```

Comment:
Is this needed? `numberOfDeathsSinceLastKill` is already being reset on line 130

Reply:
Absolutely. Here's why:

Previously:
1) Comeback triggers when let's say you die 3x, the intended way for it to work is you die 3x without scoring a kill.
2) However, due to not clearing it up when the attacker scores a kill, no matter how many times you kill, the Comeback trigger will always happen when you die for the third time.
3) After the Comeback triggers, only then does the numberOfDeathsSinceLastKill clears itself.